templates/download/mediatek-genio/tab-2.html

```diff
@@ -0,0 +1,58 @@
+<div tabindex="0" role="tabpanel" id="genio-700-evk-tab" aria-labelledby="genio-700-evk" aria-hidden="true">
+	<div class="row p-strip is-shallow u-no-padding--top">
+    <div class="col-3">
+      <h2 class="u-hide--small">Genio 700 EVK</h2>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/bb347ce4-hw_evk_g700-evk_isometric.jpg",
+        alt="",
+        width="1828",
+        height="2188",
+        hi_def=True,
+        loading="auto",
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
+    <div class="col-3">
+      <h3 class="p-heading--5 u-no-padding--top">Ubuntu Desktop 22.04 LTS</h3>
+    </div>
+    <div class="col-6">
+      <p class="u-no-padding--top">The version of ubuntu with up to 10 years of long term support, until april 2032.</p>
+      <hr class="u-no-margin--bottom">
+      <div class="p-notification--information is-borderless">
+        <div class="p-notification__content">
+          <p class="p-notification__message">Please download boot firmware before you start your installation.</p>
+        </div>
+      </div>
+      <p>
+				<a class="p-button--positive" href="https://people.canonical.com/~platform/images/mediatek/ubuntu-desktop-22.04/genio-classic-desktop-2204-x01-20231005-133.tar.xz">Download 22.04 LTS</a>
```

Comment:
Juan suggested in the copy docs `Download Ubuntu Desktop 22.04 LTS` for accessibility reasons